network/src/inbound/inbound.rs

```diff
@@ -77,36 +73,19 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
             loop {
                 match listener.accept().await {
                     Ok((stream, remote_address)) => {
-                        info!("Got a connection request from {}", remote_address);
-
                         if !node_clone.can_connect() {
-                            metrics::increment_counter!(stats::CONNECTIONS_ALL_REJECTED);
```

Comment:
why is this counter removed? not accepting a connection due to connection limits is something we'd like to know about, as it could indicate that the limit is too low